skrub/_reporting/_data/templates/base.css

```diff
@@ -251,6 +256,10 @@ dd {
     cursor: pointer;
 }
 
+.float-left {
```

Comment:
why do we float the bar & buttons? it results in wasting space and looks a bit weird IMO

![screenshot_2025-03-19T19:10:05+01:00](https://github.com/user-attachments/assets/bda42ab0-43db-4802-b7ce-ff434f2dd593)

Reply:
also moving the buttons to the right means there is much less space for the tooltip

![screenshot_2025-03-19T19:33:52+01:00](https://github.com/user-attachments/assets/d8301b19-99af-4d46-96d1-d3010aa7cca9)


which was one reason for having it on the left

Reply:
Seriously, vertical space.

Look, this is the user experience, even with the button on this side
![image](https://github.com/user-attachments/assets/894206a1-700a-4130-a236-d4b6eec1bef2)

The table report takes a full screen. I cannot see the line that I have run to generate it. It's really annoying. It's a major usability problem, and TBH a reason for me not wanting to use it.

We can improve the tooltip, but it really does not matter compared the waste of vertical space

Reply:
Also, I think that the problem that you flagged cannot happen with the new code that I have committed.